src/detect-engine-uint.c

```diff
@@ -373,6 +454,10 @@ DetectU8Data *DetectU8Parse (const char *u8str)
             return NULL;
         }
     }
+    if (DetectU8Validate(&u8da)) {
+        SCLogError(SC_ERR_INVALID_VALUE, "Impossible value for uint8 condition");
```

Comment:
can we provide a more helpful error here? Something that includes the keyword name and perhaps also the original value?

Reply:
Ok for the original value.
For the keyword name, should it be the caller of `DetectU32Parse` (such as `DetectHTTP2windowSetup`) which should print another message or should it transmit the keyword name to `DetectU32Parse` just for this debugging purpose ?